app/app.py

```diff
@@ -5,41 +5,63 @@
 from flask            import Flask
 from flask_sqlalchemy import SQLAlchemy
 
+
 app = Flask(__name__)
+db = SQLAlchemy(app)
 
 app.config.from_object('config')
 
 # Please use a sane timezone for log entries so that we don't have to jump
 # through daylight savings hoops.
 logging.Formatter.converter = time.gmtime
-formatter = logging.Formatter('%(asctime)s [%(levelname)s] %(module)s,l%(lineno)d : %(message)s')
+formatter = logging.Formatter('%(asctime)s [%(levelname)s] %(threadName)s,%(module)s,l%(lineno)d : %(message)s')
 
-logger = app.logger
+# Set up the level at which we want to log
+if app.config['TESTING'] or app.config['DEBUG']:
+    level = logging.DEBUG
+else:
+    level = logging.INFO
 
-if app.config['DEBUG'] == True:
-    # Log to the console when in debug/test mode
-    cons_log = logging.StreamHandler()
-    cons_log.setLevel(logging.DEBUG)
-    cons_log.setFormatter(formatter)
-    logger.addHandler(cons_log)
-    # Also log to a local file, rather than to the system log dir
+loggers = {app.logger                      : None,
+           logging.getLogger('werkzeug')   : logging.INFO,
+           logging.getLogger('sqlalchemy') : logging.WARN}
+handlers = []
+
+# Set up the log handlers.
+if app.config['TESTING'] or app.config['DEBUG']:
```

Comment:
The decision being made is whether we're running in a local dev environment, or in production.  Even though I'd expect that we'd want debug logging when in a local dev environment, logging at debug isn't synonymous with running in a local env.  

I can see that I should have wrapped the check of the two variables into a function (something like `islocalenv`); would that address your concern?